code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -302,6 +302,9 @@
 	default_unfasten_wrench(user, I, 4 SECONDS)
 
 /obj/machinery/chem_dispenser/attack_ai(mob/user)
+	if(isdrone(user))
+		// There's nothing a drone can do here that wouldn't violate their laws and/or the rules.
```

Comment:
There should probably be a to_chat here explaining to the drone they cant use a chem dispenser, probably with a warning span class